src/components/App.jsx

```diff
@@ -13,8 +13,9 @@ import injectTapEventPlugin from 'react-tap-event-plugin'
 injectTapEventPlugin()
 
 Form.addInputTypes({
-  string: BNCTextField,
-  phone: BNCPhoneField
+  email: BNCTextField,
+  text: BNCTextField,
```

Comment:
This part actually shouldn't be necessary.  The point of the 'inputTypes' thing here is to map React components to input types used by react-formal.  Unless we plan to have different React components handling text vs phone, we don't need to change this dict.